SpatialGDK/Source/SpatialGDK/Private/SpatialView/SubView.cpp

```diff
@@ -0,0 +1,217 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialView/SubView.h"
+
+#include "Utils/ComponentFactory.h"
+
+namespace SpatialGDK
+{
+SubView::SubView(const Worker_ComponentId TagComponentId, const FFilterPredicate Filter, const EntityView& View, FDispatcher& Dispatcher,
```

Comment:
Comes up in a few places - I think some of these should be by const ref instead of by value. Can comment on each one if helpful.

Reply:
const ref'd everything in the new PR https://github.com/spatialos/UnrealGDK/pull/2522/files